.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: objective-c
+os: osx
+osx_image: xcode8.3
```

Comment:
The PR looks good. Just want to know if the script also works in new xcode9? If yes, maybe using xcode9.1 image is better.:)

Reply:
Agreed. Thanks for suggesting this.